src/com/infected/util/Pause.java

```diff
@@ -8,4 +8,11 @@ public static void pause() {
             e.printStackTrace();
         }
     }
+    public static void pause2() {
```

Comment:
instead of creating a 2nd duplicate method. you can use existing method but change the signature to pass in a parameter for milliseconds